packages/features/schedules/lib/use-schedule/useSchedule.ts

```diff
@@ -114,13 +116,33 @@ export const useSchedule = ({
       (Boolean(eventSlug) || Boolean(eventId) || eventId === 0),
   };
 
-  let schedule;
+  const teamSchedule = useApiV2AvailableSlots({
+    enabled: Boolean(isTeamEvent),
+    ...input,
+    duration: input.duration ? Number(input.duration) : undefined,
+    routedTeamMemberIds: input.routedTeamMemberIds ?? undefined,
+    teamMemberEmail: input.teamMemberEmail ?? undefined,
+    eventTypeId: eventId ?? undefined,
+  });
+
   if (isTeamEvent) {
-    schedule = trpc.viewer.highPerf.getTeamSchedule.useQuery(input, options);
-  } else {
-    schedule = trpc.viewer.slots.getSchedule.useQuery(input, options);
+    updateEmbedBookerState({
+      bookerState,
+      slotsQuery: teamSchedule,
+    });
```

Comment:
FYI @hariombalhara I had to add another instance of this inside this block now due to the refactor we do in this PR